cmd/manager/main.go

```diff
@@ -116,7 +119,21 @@ func main() {
 	register3scaleVersionInfoMetric()
 
 	// Create Service object to expose the metrics port.
-	_, err = metrics.ExposeMetricsPort(ctx, metricsPort)
+	servicePorts := []v1.ServicePort{
```

Comment:
@sergioifg94 The migration guide is not good at all explaining the changes needed to this file. Additional changes should be made here.

You should look at the scaffolding from the operator-sdk - https://github.com/operator-framework/operator-sdk/blob/v0.14.1/internal/scaffold/cmd.go#L39-L225

Take a look at my recent PR to the Enmasse operator as well - https://github.com/EnMasseProject/enmasse/pull/3716